src/DocumentFormat.OpenXml/Packaging/ThumbnailPartExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DocumentFormat.OpenXml.Packaging
+{
+    internal static partial class ThumbnailPartExtensions
```

Comment:
These extensions should be public. Also, can you put them all in a single class for now? Along with the new unit method. We may move them later but it'll be easier to review and make sure we have consistemcy